src/release.js

```diff
@@ -551,6 +551,44 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
             }
         },
 
+
+        /**
+         * Merge release branch into releasing branch and ask user to resolve conflicts manually if any
+         */
+        async mergeWithReleaseBranch() {
+            log.doing(`Merging '${releaseBranch}' into '${data.releasingBranch}'.`);
+
+            try {
+                // checkout master
+                await gitClient.checkout(releaseBranch);
+
+                // pull master
+                await gitClient.pull(releaseBranch);
+
+                // checkout releasingBranch
+                await gitClient.checkout(`${branchPrefix}-${data.version}`);
+
+                // merge release branch into releasingBranch
+                await gitClient.merge([releaseBranch]);
+
+                log.done(`'${releaseBranch}' merged into '${branchPrefix}-${data.version}'.`);
+            } catch (err) {
+                log.warn('Please resolve the conflicts and complete the merge manually (including making the merge commit).');
+
+                const mergeDone = await this.promptToResolveConflicts();
+                if (mergeDone) {
+                    if (await gitClient.hasLocalChanges()) {
+                        log.exit(`Cannot push changes because local branch '${data.releasingBranch}' still has changes to commit.`);
+                    }
+                    await gitClient.push(origin, data.releasingBranch);
+                    log.done(`'${releaseBranch}' merged into '${branchPrefix}-${data.version}'.`);
+                    return;
```

Comment:
To stay in line with the coding style of the project, I would remove the early `return;`
You could achieve it with:
```
if (mergeDone) {
    //...
}
else {
    await gitClient.abortMerge([releaseBranch]);
    log.exit();
}
```

Reply:
fine by me.